posthog/tasks/utils.py

```diff
@@ -31,6 +31,7 @@ class CeleryQueue(Enum):
     EMAIL = "email"
     LONG_RUNNING = "long_running"  # any task that has a good chance of taking more than a few seconds should go here
     ANALYTICS_QUERIES = "analytics_queries"
+    ANALYTICS_LIMITED = "analytics_limited"
```

Comment:
BTW worth having a docstring on this enum saying `Look at config/posthog/prod-{us,eu}.yaml.gotmpl`, since I actually had no idea where these queues are defined, and now I do thanks to your https://github.com/PostHog/charts/pull/1499/files link 😀

Reply:
Isn't this what the main comment at the top of the file is for 😅